dvc/repo/scm_context.py

```diff
@@ -3,7 +3,11 @@ def run(repo, *args, **kw):
         try:
             result = method(repo, *args, **kw)
             repo.scm.reset_ignores()
-            repo.scm.remind_to_track()
+            auto_stage = repo.config.get("core", {}).get("autostage", False)
+            if auto_stage:
```

Comment:
Just for consistency :slightly_smiling_face: 
```suggestion
            autostage = repo.config.get("core", {}).get("autostage", False)
            if autostage:
```